### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by RealRichi3** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#15] [Outreachy 2022] [answered]
Hello everyone, I am Richie Moluno, I am one of the Outreachy interns. I look forward to learning from everyone and making meaningful contributions to this project

**Comment by BatoolMM** [comment, marked as answer]
Welcome @RealRichi3 to the community!

We have two projects for Outreachy, you can find them here:
1- [Improve the bilinguality and accessibility of the website](https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website/issues/12)
2- Improve Localisation for [Open Educational Resources](https://github.com/Open-Science-Community-Saudi-Arabia/localisation/issues/13).

Feel free to comment under the issue, if you have any questions!

**Comment by yinkabasit** [comment]
Hello, I am Abdulbasit Olayinka from Nigeria. . I have picked to contribute to the OSCSA project because it interest me. How can I get started to contribute . Thank you

**Reply by BatoolMM** [reply]
Welcome @yinkabasit!
You can head to the issue we create and start to pick a task (task mentioned above). After you pick the task, you can create your PR.
